pytest/lib/transaction.py

```diff
@@ -0,0 +1,167 @@
+            ['keyType', 'u8'],
+            ['data', [32]]
+        ] }],
+[AccessKey, { 'kind': 'struct', 'fields': [
+            ['nonce', 'u64'],
+            ['permission', AccessKeyPermission],
+        ] }],
+[AccessKeyPermission, { 'kind': 'enum', 'field': 'enum', 'values': [
+            ['functionCall', FunctionCallPermission],
+            ['fullAccess', FullAccessPermission],
+        ] }],
+[FunctionCallPermission, { 'kind': 'struct', 'fields': [
+            ['allowance', { 'kind': 'option', type: 'u128' }],
+            ['receiverId', 'string'],
+            ['methodNames', ['string']],
+        ] }],
+[FullAccessPermission, { 'kind': 'struct', 'fields': [] }],
+[Action, { 'kind': 'enum', 'field': 'enum', 'values': [
+            ['createAccount', CreateAccount],
+            ['deployContract', DeployContract],
+            ['functionCall', FunctionCall],
+            ['transfer', Transfer],
+            ['stake', Stake],
+            ['addKey', AddKey],
+            ['deleteKey', DeleteKey],
+            ['deleteAccount', DeleteAccount],
+        ] }],
+[CreateAccount, { 'kind': 'struct', 'fields': [] }],
+[DeployContract, { 'kind': 'struct', 'fields': [
+            ['code', ['u8']]
+        ] }],
+[FunctionCall, { 'kind': 'struct', 'fields': [
+            ['methodName', 'string'],
+            ['args', ['u8']],
+            ['gas', 'u64'],
+            ['deposit', 'u128']
+        ] }],
+[Transfer, { 'kind': 'struct', 'fields': [
+            ['deposit', 'u128']
+        ] }],
+[Stake, { 'kind': 'struct', 'fields': [
+            ['stake', 'u128'],
+            ['publicKey', PublicKey]
+        ] }],
+[AddKey, { 'kind': 'struct', 'fields': [
+            ['publicKey', PublicKey],
+            ['accessKey', AccessKey]
+        ] }],
+[DeleteKey, { 'kind': 'struct', 'fields': [
+            ['publicKey', PublicKey]
+        ] }],
+[DeleteAccount, { 'kind': 'struct', 'fields': [
+            ['beneficiaryId', 'string']
+        ] }],
+])
+
+def sign_and_serialize_transaction(receiverId, nonce, actions, blockHash, accountId, pk, sk):
+    tx = Transaction()
+    tx.signerId = accountId
+    tx.publicKey = PublicKey()
+    tx.publicKey.keyType = 0
+    tx.publicKey.data = pk
+    tx.nonce = nonce
+    tx.receiverId = receiverId
+    tx.actions = actions
+    tx.blockHash = blockHash
+
+    msg = BinarySerializer(tx_schema).serialize(tx)
+    hash_ = hashlib.sha256(msg).digest()
+
+    signature = Signature()
+    signature.keyType = 0
+    signature.data = SigningKey(sk).sign(hash_)
+
+    signedTx = SignedTransaction()
+    signedTx.transaction = tx
+    signedTx.signature = signature
+
+    return BinarySerializer(tx_schema).serialize(signedTx)
+
+def create_payment_action(amount):
+    transfer = Transfer()
+    transfer.deposit = amount
+    action = Action()
+    action.enum = 'transfer'
+    action.transfer = transfer
+    return action
+
+def sign_payment_tx(validator, to, amount, nonce, blockHash):
```

Comment:
account here instead of validator? and just do `validator.account` where you calling